yarn.lock

```diff
@@ -7331,8 +7331,8 @@ matrix-mock-request@^1.2.3:
     counterpart "^0.18.6"
     diff-dom "^4.2.2"
     diff-match-patch "^1.0.5"
-    emojibase-data "^6.2.0"
-    emojibase-regex "^5.1.3"
+    emojibase-data "^5.1.1"
+    emojibase-regex "^4.1.1"
```

Comment:
unrelated?

Reply:
Indeed, what on earth are you doing yarn

Reply:
being yarn